src/ck-core/ckcallback.h

```diff
@@ -77,12 +78,16 @@ class CkCallback {
       case bcastArray: return "CkCallback::bcastArray";
       case bcastSection: return "CkCallback::bcastSection";
       case replyCCS: return "CkCallback::replyCCS";
+	  case sendFuture: return "CkCallback::sendFuture";
```

Comment:
Indentation doesn't match surrounding.

Reply:
I agree that we should fix the formatting.